chainerx_cc/chainerx/routines/math.cc

```diff
@@ -1140,4 +1140,84 @@ Array IsFinite(const Array& x) {
     return out;
 }
 
+template <typename Impl>
+inline Array BitwiseImpl(Impl&& impl, const Array& x1, const Array& x2) {
+    CHAINERX_ASSERT(GetKind(x1.dtype()) != DtypeKind::kFloat);
```

Comment:
Could you raise `DtypeError`?
`CHAINERX_ASSERT` is used only for the test, and does not expose errors to Python.

Reply:
Oh, sure